by-language/java-jooq/README.rst

```diff
@@ -35,24 +35,48 @@ In some kind, jOOQ is similar to `LINQ`_, `but better <Insight into Language
 Integrated Querying_>`_.
 
 
+*******
 Details
-=======
+*******
+
+Overview
+========
+
+The code examples will demonstrate a few of the `different use cases for jOOQ`_.
+That is, how to use the `jOOQ DSL API`_ based on code generated with `jOOQ's
+code generator`_ to take your database schema and reverse-engineer it into a
+set of Java classes, as well how to use the `Dynamic SQL API`_ by using `jOOQ
+as a SQL builder without code generation`_.
 
-The code example will demonstrate a few of the `different use cases for jOOQ`_.
-That is, `Dynamic SQL`_, the `jOOQ DSL API`_, and how to use `jOOQ as a SQL
-builder without code generation`_.
+Schema management
+=================
 
+In many cases, the schema is defined in the form of SQL scripts, which can be
+used with a `database schema migration`_ framework like `Flyway`_,
+`Liquibase`_, `Bytebase`_, etc.
+
+The `DDLDatabase - Code generation from SQL files`_ feature can be used to
+effectively reflect the database schema from SQL DDL files, without needing
+a database instance at all. The code provided within the ``src/generated``
+directory has been generated like this.
 
 Caveats
 =======
 
-- `jOOQ's code generator`_ takes your database schema and reverse-engineers it
-  into a set of Java classes. This feature currently does not work with CrateDB
-  yet. The code provided within the ``src/generated`` directory has not been
-  derived by reflecting the database schema from CrateDB.
+- `jOOQ's code generator`_ currently does not work with directly connecting to
```

Comment:
I would probably mention that since we are using the psql jdbc driver and we are bound to JOOQ parser, we can only use sql-standard/psql based definitions.

Reply:
I've improved the "Caveats" section with 15ae4cd5c9ea5365c. Thank you again.